### PR TITLE
[Audit Fix][N-05] Fix inconsistent event emission parameters

### PR DESCRIPTION
**Pull request by haythem96**
# Task: Fix inconsistent event emission parameters

**Should be merged after #780**

## Description

Fixes ENG-1024

## Type of change

- [ ] New feature
- [X] Bug fix
- [ ] Testing code
- [ ] Document update or config files

### CODE REVIEW

File: packages/bull-vault/src/BullStrategy.sol [hunk 1]
```diff
@@ -137,7 +137,7 @@ contract BullStrategy is ERC20, LeverageBull {
     function setShutdownContract(address _shutdownContract) external onlyOwner {
         require(_shutdownContract != address(0), "BS6");
 
-        emit SetShutdownContract(shutdownContract, _shutdownContract);
+        emit SetShutdownContract(_shutdownContract, shutdownContract);
```
Comment (aleone) [inline packages/bull-vault/src/BullStrategy.sol:140]: this is now backwards _shutdownContract is the new one, and shutdownContract is the old one

**Review by aleone — APPROVED** [review]
one small change needed (reversed shutdown contract order in event), but approving

**Review by alpinechicken — APPROVED** [review]
lgtm

File: packages/bull-vault/src/AuctionBull.sol [hunk 2]
```diff
@@ -169,7 +169,17 @@ contract AuctionBull is UniFlash, Ownable, EIP712 {
...
+
+    /**
+     * @notice constructor for AuctionBull
+     * @param _auctionManager the address that can run auctions
+     * @param _bull bull strategy address
+     * @param _factory uniswap factory address
+     * @param _crab crab strategy address
+     * @param _eToken euler collateral token address for weth
+     * @param _dToken euler debt token address for usdc
+     */
```
Comment (aleone) [inline packages/bull-vault/src/AuctionBull.sol:183]: i think we got some weird merge stuff here, why 3 natspec repeated?

**Pull request merged**